chaosblade-exec-common/src/main/java/com/alibaba/chaosblade/exec/common/center/DefaultPluginBeanManager.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 1999-2019 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.chaosblade.exec.common.center;
+
+import com.alibaba.chaosblade.exec.common.aop.Plugin;
+import com.alibaba.chaosblade.exec.common.aop.PluginBean;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DefaultPluginBeanManager implements PluginBeanManager {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultPluginBeanManager.class);
+
+    private Map<String, PluginBean> plugins = new HashMap<String, PluginBean>();
+
+    @Override
+    public void load() {
+    }
+
+    @Override
+    public void unload() {
+        plugins.clear();
+    }
+
+    @Override
+    public PluginBean getPlugin(String target) {
+        return plugins.get(target);
+    }
+
+    @Override
+    public void registerPlugin(PluginBean plugin) {
+        String target = plugin.getModelSpec().getTarget();
```

Comment:
There may be different pluginbeans, but the modelSpec is the same. For example, Dubbo has provider and consumer pluginBean, but The name of modelSpec is both Dubbo

Reply:
This will change later, do not merge

Reply:
done